src/chis_ne/res/values/strings.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+	<string name="app_name">Community Health Information System</string>
```

Comment:
I would expect a shorter and more specific name here, this string is used as a name for the app in Android when is listed in the apps menu. Maybe "CHT CHIS Nepal", or just "CHIS Nepal" ?

Reply:
Changed to "CHIS", thanks.